src/BuildScriptGenerator/DotNetCore/ProjectFileProviders/ProbeAndFindProjectFileProvider.cs

```diff
@@ -5,27 +5,22 @@
 
 using System.Collections.Generic;
 using System.Linq;
-using System.Reflection.Metadata;
 using Microsoft.Extensions.Logging;
 using Microsoft.Oryx.BuildScriptGenerator.Exceptions;
-using Microsoft.Oryx.BuildScriptGenerator.Resources;
-using Microsoft.Oryx.Common.Extensions;
 
 namespace Microsoft.Oryx.BuildScriptGenerator.DotNetCore
 {
     internal class ProbeAndFindProjectFileProvider : IProjectFileProvider
     {
         private readonly ILogger<ProbeAndFindProjectFileProvider> _logger;
-        private readonly IStandardOutputWriter _writer;
 
         // Since this service is registered as a singleton, we can cache the lookup of project file.
         private bool _probedForProjectFile;
         private string _projectFileRelativePath;
 
-        public ProbeAndFindProjectFileProvider(ILogger<ProbeAndFindProjectFileProvider> logger, IStandardOutputWriter writer)
+        public ProbeAndFindProjectFileProvider(ILogger<ProbeAndFindProjectFileProvider> logger)
```

Comment:
Why do we not keep the output writer to alert users in this part

Reply:
This is because we have received feedback from jamstack team about their users complaining about this message which it seem is confusing for them (as jamstack only supports node + some static tools based apps). Also we support `.fsproj` apart from `.csproj` files, so instead of adding more confusing logs to existing issue, I am adding a general message like `Detecting platforms...` so that it is is applicable to all scenarios.